modules/extensions-support/src/test/java/org/mule/module/extension/internal/manager/DefaultExtensionManagerTestCase.java

```diff
@@ -61,6 +61,8 @@
 public class DefaultExtensionManagerTestCase extends AbstractMuleTestCase
```

Comment:
add tests that:

* Verify you can't have a null vendor
* Extensions of same name but different vendor are not consider equal
* Exensions of same name and vendor are consider equal